akka-http-core/src/main/resources/reference.conf

```diff
@@ -233,6 +233,16 @@ akka.http {
       tcp-no-delay = undefined
     }
 
+    # Client https proxy options. When using ClientTransport.httpsProxy() with or without credentials,
+    # host/port must be either passed explicitly or set here. When using the proxy from settings leaving these set as
+    # undefined/-1, an error will be thrown indicating that the options must be set.
+    proxy {
+      https {
+        host = undefined
+        port = -1
```

Comment:
Hmm, we don't have a great way to encode 'by default unspecified' settings... could we find a default? For the port perhaps 443 might make sense as a default, maybe use the empty string for an unspecified host?